src/lib/user_command_input/dune

```diff
@@ -0,0 +1,9 @@
+(library
```

Comment:
Can this file just be part of `coda_base` rather than being it's own library? I'm typically not a fan of all the micro-libraries we have, but I don't know how others feel about it.

Reply:
I did that because there's dependency cycle between `coda_base` and `secrets`